 ## Check der Daten aus dem Scraping 2020
 Das Scraping vom 14./15.1.2020 ergab 114799 Dateien (medreg_2020_scraped_data). Eine erste Analsyse ergab:
 
- 108463 Einträge mit Code 200
- 6314 Einträge mit Code 500

Aber leider wissen wir beim Error 500 nicht genau, was die Ursache war. Das Problem ist ja, dass nicht alle ID's einen Eintrag haben. Ruft der Scraper eine solche Seite auf, kommt ein Error 500 zurück. Allerdings könnte der Fehler 500 auch andere Ursachen haben.

Gemäss unserem Screenshot der MedReg-Hauptseite waren am 14.1.2020 folgende Anzahl Einträge in der Datenbank:

Ärztin/Arzt 76765
Chriropraktor 386
Zahnarzt 11997
Apotheker 10999
Tierarzt 5315
**Total: 105462**

<img src = "medreg_scraping_protokoll/Screenshot_MedReg_20200114_085447.jpg" style=width:800px/>


Es kann sich bei der Differenz um Personen handeln mit zwei Berufen. Ein Check ergibt zum Beispiel 631 Ärzte, die auch als Zahnärzte verzeichnet sind (52 als Apotheker, 18 Chiropraktoren, 8 Tierärzte).

Weiterer Checks: Eine Inhaltssuche mit der App EasyFind in den Files nach dem Begriff Ärztin/Arzt ergibt 74279 Files. Das sind 2486 weniger, als die Homepage angibt. Die Suche nach Tierärztin ergibt 5059 Treffer, 256 weniger als angegeben. Die Suche in MacOS (Spotlight) nach Psychiatrie und Psychotherapie ergibt 5209 Treffer. 5425 müssten es laut der Zählung auf der Übersichtsseite der Datenbank sein. Auch EasyFind findet 5209 Einträge in den Files.

Der erste Eintrag findet sich mit ID 552, der letzte mit 113984, ein Psychiater aus Frankreich, dessen Diplom am 15.1.2020 anerkannt wurde.

Etwas beunruhigend ist die Tatsache, dass eine Stichprobe der Files zwei unterschiedliche Dateien zeigte, die mit Code 500 registriert wurden. 
Eine Art der Files ist eine Fehlermeldung, abgelegt als html-Code. Von dieser Fehlermeldung zählen wir mit EasyFind (Suche: ``Es ist ein unerwarteter Fehler aufgetreten``) insgesamt: 6315 Files. Das entspricht bis auf 1 unseren Code 500-Fehlermeldungen. Diese Files haben folgenden Inhalt:




<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="de">
<head>
	<meta http-equiv="Content-Type" content="text/xml; charset=UTF-8" />
	<title>Fehler Erreur errore</title>
	<link href="/Content/site.css" rel="stylesheet" type="text/css" />
	<!--[if lte IE 7]>
	<link href="/Content/ie.css" rel="stylesheet" type="text/css" />
	<![endif]-->
</head>
<body>
	<div id="page">
		<div id="pageHeader">
			<div class="left"></div>
			<div class="right">
				<div class="appTitle">Bundesverwaltung admin.ch</div>
				<div class="appTitleEdi">Eidgenössisches Departement des Innern EDI</div>
				<div class="appTitleBag">Bundesamt für Gesundheit BAG</div>
			</div>
		</div>
		<div id="clearHeader"></div>
		<div id="pageNavigation">
			<div class="navigation">
				<ul>
					<li><a href="http://www.medreg.admin.ch/" title="Startseite">Startseite</a></li>
				</ul>
			</div>
		</div>
		<div id="pageContent">
			<h1>Fehler</h1>
			<div id="content">
				Es ist ein unerwarteter Fehler aufgetreten. Une erreur est survenue. Si è verificato un errore imprevisto.
			</div>
		</div>
		<div class="clearBoth"></div>
		<div id="pageFooter">
			<hr />
			<div>Bundesamt für Gesundheit (BAG)</div>
			<hr />
		</div>
	</div>
</body>
</html>


Wir finden noch weitere Files ohne Inhalt, mit dem Hinweis: *Please enable JavaScript to view the page content.* 

Hier ist der Code dieser Files (``Doppelklicken, da Code versteckt``)

<!DOCTYPE html>
<html><head>
<meta http-equiv="Pragma" content="no-cache"/>
<meta http-equiv="Expires" content="-1"/>
<meta http-equiv="CacheControl" content="no-cache"/>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<link rel="shortcut icon" href="data:;base64,iVBORw0KGgo="/>

<script>

(function(){
    var securemsg;
    var dosl7_common;

window["bobcmn"] = "111110111110102000000022000000052000000002531f809e200000096300000000300000000300000006/TSPD/300000008TSPD_101300000005https200000000200000000";

window.BVYp=!!window.BVYp;try{(function(){try{var OO,zO,sO=1,IO=1,JO=1,lO=1,LO=1,_z=1,Iz=1,lz=1,Lz=1;for(var oz=0;oz<zO;++oz)sO+=2,IO+=2,JO+=2,lO+=2,LO+=2,_z+=2,Iz+=2,lz+=2,Lz+=3;OO=sO+IO+JO+lO+LO+_z+Iz+lz+Lz;window.JI===OO&&(window.JI=++OO)}catch(zZ){window.JI=OO}var LZ=!0;function Os(O){!O||document.visibilityState&&"visible"!==document.visibilityState||(LZ=!1,document.cookie="brav=ad");return LZ}function zs(){}Os(window[zs.name]===zs);Os("function"!==typeof ie9rgb4);
Os(/\x3c/.test(function(){return"\x3c"})&!/x3d/.test(function(){return"'x3'+'d';"}));var Zs=window.attachEvent||/mobi/i.test(window["\x6e\x61vi\x67a\x74\x6f\x72"]["\x75\x73e\x72A\x67\x65\x6et"]),ss=+new Date+6E5,_s,is,Is,Js=setTimeout,ls=Zs?3E4:6E3;function os(){if(!document.querySelector)return!0;var O=+new Date,s=O>ss;if(s)return Os(!1);s=is&&!Is&&_s+ls<O;s=Os(s);_s=O;is||(is=!0,Js(function(){is=!1},1));return s}os();
document.addEventListener&&document.addEventListener("visibilitychange",function(O){document.visibilityState&&("hidden"===document.visibilityState&&O.isTrusted?Is=!0:"visible"===document.visibilityState&&(_s=+new Date,Is=!1,os()))});var OS=[17795081,27611931586,1558153217];function zS(O){O="string"===typeof O?O:O.toString(36);var s=window[O];if(!s.toString)return;var _=""+s;window[O]=function(O,_){is=!1;return s(O,_)};window[O].toString=function(){return _}}for(var Lz=0;Lz<OS.length;++Lz)zS(OS[Lz]);
Os(!1!==window.BVYp);
(function(){var O=-1,O={L:++O,LJ:"false"[O],Z:++O,__:"false"[O],iZ:++O,s2:"[object Object]"[O],ii:(O[O]+"")[O],I_:++O,_i:"true"[O],sZ:++O,JZ:++O,oJ:"[object Object]"[O],l:++O,zZ:++O,JiZ:++O,jiZ:++O};try{O.S_=(O.S_=O+"")[O.JZ]+(O.oS=O.S_[O.Z])+(O.lJ=(O.LS+"")[O.Z])+(!O+"")[O.I_]+(O.OS=O.S_[O.l])+(O.LS="true"[O.Z])+(O.Ii="true"[O.iZ])+O.S_[O.JZ]+O.OS+O.oS+O.LS,O.lJ=O.LS+"true"[O.I_]+O.OS+O.Ii+O.LS+O.lJ,O.LS=O.L[O.S_][O.S_],O.LS(O.LS(O.lJ+'"\\'+O.Z+O.JZ+O.Z+O.LJ+"\\"+O.sZ+O.L+"("+O.OS+"\\"+O.Z+O.zZ+
O.Z+"\\"+O.Z+O.l+O.L+O._i+O.oS+O.LJ+"\\"+O.sZ+O.L+"\\"+O.Z+O.l+O.zZ+"\\"+O.Z+O.JZ+O.Z+"\\"+O.Z+O.JZ+O.l+O.ii+O.oS+"\\"+O.Z+O.l+O.zZ+"['\\"+O.Z+O.l+O.L+O.__+"\\"+O.Z+O.zZ+O.Z+"false"[O.iZ]+O.oS+O.__+O.ii+"']\\"+O.sZ+O.L+"===\\"+O.sZ+O.L+"'\\"+O.Z+O.l+O.I_+O.OS+"\\"+O.Z+O.l+O.iZ+"\\"+O.Z+O.JZ+O.Z+"\\"+O.Z+O.JZ+O.l+"\\"+O.Z+O.sZ+O.zZ+"')\\"+O.sZ+O.L+"{\\"+O.Z+O.iZ+"\\"+O.Z+O.Z+"\\"+O.Z+O.l+O.l+O.__+"\\"+O.Z+O.l+O.iZ+"\\"+O.sZ+O.L+O._i+O.ii+"\\"+O.Z+O.l+O.l+O.oJ+"\\"+O.Z+O.zZ+O.Z+O.Ii+"\\"+O.Z+O.JZ+O.iZ+
"\\"+O.Z+O.JZ+O.I_+"\\"+O.Z+O.l+O.L+"\\"+O.sZ+O.L+"=\\"+O.sZ+O.L+"\\"+O.Z+O.l+O.zZ+"\\"+O.Z+O.JZ+O.Z+"\\"+O.Z+O.JZ+O.l+O.ii+O.oS+"\\"+O.Z+O.l+O.zZ+"['\\"+O.Z+O.l+O.L+O.__+"\\"+O.Z+O.zZ+O.Z+"false"[O.iZ]+O.oS+O.__+O.ii+"'].\\"+O.Z+O.l+O.iZ+O._i+"\\"+O.Z+O.l+O.L+"false"[O.iZ]+O.__+O.oJ+O._i+"(/.{"+O.Z+","+O.sZ+"}/\\"+O.Z+O.sZ+O.zZ+",\\"+O.sZ+O.L+O.LJ+O.Ii+"\\"+O.Z+O.JZ+O.l+O.oJ+O.OS+"\\"+O.Z+O.JZ+O.Z+O.oS+"\\"+O.Z+O.JZ+O.l+"\\"+O.sZ+O.L+"(\\"+O.Z+O.zZ+O.L+")\\"+O.sZ+O.L+"{\\"+O.Z+O.iZ+"\\"+O.Z+O.Z+
"\\"+O.Z+O.Z+"\\"+O.Z+O.Z+"\\"+O.Z+O.l+O.iZ+O._i+O.OS+O.Ii+"\\"+O.Z+O.l+O.iZ+"\\"+O.Z+O.JZ+O.l+"\\"+O.sZ+O.L+"(\\"+O.Z+O.zZ+O.L+"\\"+O.sZ+O.L+"+\\"+O.sZ+O.L+"\\"+O.Z+O.zZ+O.L+").\\"+O.Z+O.l+O.I_+O.Ii+O.s2+"\\"+O.Z+O.l+O.I_+O.OS+"\\"+O.Z+O.l+O.iZ+"("+O.iZ+",\\"+O.sZ+O.L+O.sZ+")\\"+O.Z+O.iZ+"\\"+O.Z+O.Z+"\\"+O.Z+O.Z+"});\\"+O.Z+O.iZ+"}\\"+O.Z+O.iZ+'"')())()}catch(s){O%=5}})();var ZS=71;window.Sl={Il:"084f6d8d0c0178009c3cf6d52ca2f7a82bf544d91f554158432b4674244008be4520ef8f084f9409c4cafb7091547122cac37a1619d227a096762ed0ec17255d493c119249341d6c634dea57361c22d12f7577c6d4be0db9539652afc2d915fac21492028de9887b3ad36a846b1f065b229562f095b892d4a4db953aa82f6579"};function z(O){return 353>O}
function Z(O){var s=arguments.length,_=[];for(var J=1;J<s;++J)_.push(arguments[J]-O);return String.fromCharCode.apply(String,_)}function S(O,s){O+=s;return O.toString(36)}(function(O){O||setTimeout(function(){if(!os())return;var O=setTimeout(function(){},250);for(var _=0;_<=O;++_)clearTimeout(_);os()},500)})(os());})();}catch(x){document.cookie='brav=oex'+x;}finally{ie9rgb4=void(0);};function ie9rgb4(a,b){return a>>b>>0};

})();
</script>
<script type="text/javascript" src="/TSPD/08fe5c3c79ab2000fcc426e4ab16ed2cf2637ee400839c8b8a19879e0179cae2d93802d956e0bd59?type=8"></script>
<script>
(function(){
    var securemsg;
    var dosl7_common;

window["blobfp"] = "1111111110112000003e8238f5d2bf30000004aabe743851b302d5f005e1f2c7800000e10ca548cab5ca8859899f63d01424dff6403f1514c300000020http://re.security.f5aas.com/re/";

})();
</script>
<script type="text/javascript" src="/TSPD/08fe5c3c79ab2000fcc426e4ab16ed2cf2637ee400839c8b8a19879e0179cae2d93802d956e0bd59?type=11"></script>
<noscript>Please enable JavaScript to view the page content.</noscript>
</head><body>
</body></html>

Eine Suche in EasyFind findet **8345** solcher Files. (BTW: MacOS-Spotlight-Suche nach Content findet nur 285 !)

Was ist damit?
Eine Stichprobe mit dem Scraping-Protokoll zeigt: diese Files wurden mit Code 200 antworten registriert. 

Wir machen kurz einen manuellen Test auf dem Server mit einigen ausgewählten Adressen, die mit Javascript-Hinweis runtergekommen sind:



In [7]:
import requests
import random
import time
import csv
from time import sleep

url_stamm = "https://www.medregom.admin.ch/DE/Detail/Detail?pid="
output_path = "medreg_errortest/"
headers = {'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36'}

for url_id in range(30390,30409): #alle diese IDs kamen mit Javascript-Meldung zurück
    url_req = url_stamm + str(url_id) 
    try:
        r = requests.get(url_req, headers = headers, timeout=20)

        with open(output_path + "medreg_" + str(url_id) + ".html", 'w', encoding = 'utf-8') as f:
            f.write(r.text)
            f.close()
            protokoll = [[url_id, url_req, r.status_code]]
            print(protokoll)
          
        #sleep(random.uniform(2, 5))
        #im zweiten Durchgang aktiviert
   
    except: 
        r = requests.exceptions.ConnectionError
        print(url_req, 'Scraping paused')
        time.sleep(10)
        
        continue

[[30390, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30390', 500]]
[[30391, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30391', 500]]
[[30392, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30392', 500]]
[[30393, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30393', 500]]
[[30394, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30394', 500]]
[[30395, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30395', 500]]
[[30396, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30396', 500]]
[[30397, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30397', 500]]
[[30398, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30398', 500]]
[[30399, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30399', 500]]
[[30400, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30400', 200]]
[[30401, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30401', 200]]
[[30402, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30402', 200]]
[[30403, 'https://www.med

``range(114941,114962)``
1. Durchgang: Die ersten 5 Files kommen als Fehlermeldung zurück, die anderen mit dem Javascript-Hinweis. Möglicherweise läuft die Abfrage zu schnell, wenn mehrere leere Einträge hintereinander kommen. Wir bauen deshalb einen Timesleep ein (2-5 Sekunden) und lassen die Abfrage nochmals laufen. 
2. Durchgang: Holy shit, jetzt kommen sogar alle Files mit dem Javascript-Hinweis zurück. Nochmals ohne Sleep-Bremse:
3. Durchgang: wieder alle als Javascriptmeldung. Anderer Bereich?

``range(30390,30409)``
4. Durchgang: im Print schön zu sehen: bis 30399 Code 500, dann ab 30400 Code 200, aber nur Javascript. 


Wir stellen bein weiteren manuellen Check von URLs fest, dass in einem Fall auch ein echter Eintrag zum Vorschein kommt. Wir müssen also alle Javascript-Files nochmals nachscrapen. Doch wie verhindern wir, dass das MedReg wieder Javascript-Meldungen rausgibt? 
Wir versuchen es, die Daten nicht der Reihe nach abzufragen. Hier ein erster Test:

In [10]:
url_list = list(range(45874,45884))
print(url_list)

[45874, 45875, 45876, 45877, 45878, 45879, 45880, 45881, 45882, 45883]


In [11]:
import random

random.shuffle(url_list)
print(url_list)

[45880, 45881, 45875, 45877, 45883, 45879, 45874, 45878, 45882, 45876]


In [12]:
url_list = list(range(45874,45884))

random.shuffle(url_list)

url_stamm = "https://www.medregom.admin.ch/DE/Detail/Detail?pid="
output_path = "medreg_errortest/"
headers = {'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36'}

for url_id in url_list: #alle diese IDs kamen mit Javascript-Meldung zurück
    url_req = url_stamm + str(url_id) 
    try:
        r = requests.get(url_req, headers = headers, timeout=20)

        with open(output_path + "medreg_" + str(url_id) + ".html", 'w', encoding = 'utf-8') as f:
            f.write(r.text)
            f.close()
            protokoll = [[url_id, url_req, r.status_code]]
            print(protokoll)
          
        sleep(random.uniform(2, 5))
        
    except: 
        r = requests.exceptions.ConnectionError
        print(url_req, 'Scraping paused')
        time.sleep(10)
        
        continue

[[45883, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45883', 500]]
[[45878, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45878', 500]]
[[45881, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45881', 500]]
[[45876, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45876', 500]]
[[45877, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45877', 500]]
[[45879, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45879', 500]]
[[45875, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45875', 500]]
[[45880, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45880', 500]]
[[45882, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45882', 500]]
[[45874, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45874', 500]]


Dieser Ansatz könnte funktionieren. Jetzt haben wir alles korrekte Fehlermeldungen im Ordner, keine Javascriptfiles mehr. 

Nächster Schritt: wir müssen jetzt alle 6315 gefundenen Files mit Javascript-Meldung zu einer neuen Re-Scrape-Liste verarbeiten. 

1. Dazu kopieren wir die mit EasyFind gefundenen Dateien in den neuen Ordner *medreg_2020_scraped_javascript*.



In [21]:
import os
filelist = os.listdir("medreg_2020_scraped_javascript/")
print(filelist)

['medreg_114958.html', 'medreg_114942.html', 'medreg_114954.html', 'medreg_30396.html', 'medreg_30400.html', 'medreg_30401.html', 'medreg_30397.html', 'medreg_114955.html', 'medreg_114943.html', 'medreg_114959.html', 'medreg_45880.html', 'medreg_114944.html', 'medreg_114952.html', 'medreg_45879.html', 'medreg_30390.html', 'medreg_30406.html', 'medreg_114948.html', 'medreg_45875.html', 'medreg_45874.html', 'medreg_114949.html', 'medreg_30407.html', 'medreg_30391.html', 'medreg_45878.html', 'medreg_114953.html', 'medreg_114945.html', 'medreg_45881.html', 'medreg_30404.html', 'medreg_30392.html', 'medreg_114950.html', 'medreg_114946.html', 'medreg_45882.html', 'medreg_30408.html', 'medreg_45877.html', 'medreg_45876.html', 'medreg_45883.html', 'medreg_114947.html', 'medreg_114951.html', 'medreg_30393.html', 'medreg_30405.html', 'medreg_114960.html', 'medreg_30398.html', 'medreg_30402.html', 'medreg_30394.html', 'medreg_114956.html', 'medreg_114941.html', 'medreg_114957.html', 'medreg_30395

Geht ja auch ohne Regex:

In [26]:
html = [(sub.split('_')[1]) for sub in filelist] 
urlid = [int(nr.split('.')[0]) for nr in html] 
print(urlid)

[114958, 114942, 114954, 30396, 30400, 30401, 30397, 114955, 114943, 114959, 45880, 114944, 114952, 45879, 30390, 30406, 114948, 45875, 45874, 114949, 30407, 30391, 45878, 114953, 114945, 45881, 30404, 30392, 114950, 114946, 45882, 30408, 45877, 45876, 45883, 114947, 114951, 30393, 30405, 114960, 30398, 30402, 30394, 114956, 114941, 114957, 30395, 30403, 30399, 114961]


Und jetzt richtig:


In [37]:
import os

filelist = os.listdir("medreg_2020_scraped_javascript/")

html = [(sub.split('_')[1]) for sub in filelist] 
url_list = [int(nr.split('.')[0]) for nr in html] 


In [38]:
len(url_list)

8345

In [39]:
random.shuffle(url_list)

url_stamm = "https://www.medregom.admin.ch/DE/Detail/Detail?pid="
output_path = "medreg_rescrape2/"
headers = {'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36'}

for url_id in url_list: #alle diese IDs kamen mit Javascript-Meldung zurück
    url_req = url_stamm + str(url_id) 
    try:
        r = requests.get(url_req, headers = headers, timeout=20)

        with open(output_path + "medreg_" + str(url_id) + ".html", 'w', encoding = 'utf-8') as f:
            f.write(r.text)
            f.close()
            protokoll = [[url_id, url_req, r.status_code]]
            print(protokoll)
        
        
        with open("/Users/master/Desktop/CAS_DJ/MedReg/MedReg Scraping 2020/medreg_scraping_protokoll/rescrape2_protokoll.csv", 'a') as p:
            writer = csv.writer(p, dialect = 'excel')
            writer.writerows(protokoll)
        
        sleep(random.uniform(1, 5))
        
    except: 
        r = requests.exceptions.ConnectionError
        print(url_req, 'Scraping paused')
        time.sleep(120)
        
        continue

[[46519, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46519', 500]]
[[63495, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63495', 500]]
[[114790, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114790', 500]]
[[61164, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61164', 200]]
[[57307, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57307', 500]]
[[65681, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65681', 200]]
[[65655, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65655', 200]]
[[58716, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58716', 200]]
[[62586, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62586', 500]]
[[66054, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66054', 200]]
[[114713, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114713', 500]]
[[60748, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60748', 500]]
[[69889, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69889', 200]]
[[114189, 'https://ww

[[65705, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65705', 500]]
[[56448, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56448', 200]]
[[62725, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62725', 200]]
[[66130, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66130', 200]]
[[114304, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114304', 500]]
[[56726, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56726', 500]]
[[62804, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62804', 200]]
[[57138, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57138', 500]]
[[62270, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62270', 500]]
[[69950, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69950', 200]]
[[62295, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62295', 200]]
[[56125, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56125', 200]]
[[56385, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56385', 200]]
[[61034, 'https://www.m

[[79080, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79080', 200]]
[[62652, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62652', 500]]
[[62455, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62455', 200]]
[[57408, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57408', 500]]
[[114420, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114420', 500]]
[[56651, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56651', 500]]
[[60442, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60442', 500]]
[[114796, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114796', 500]]
[[57081, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57081', 200]]
[[79084, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79084', 200]]
[[63450, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63450', 200]]
[[70169, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70169', 200]]
[[114089, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114089', 500]]
[[57369, 'https://w

[[65715, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65715', 500]]
[[61394, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61394', 500]]
[[69880, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69880', 200]]
[[60948, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60948', 500]]
[[61909, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61909', 500]]
[[63524, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63524', 200]]
[[60683, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60683', 500]]
[[46492, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46492', 200]]
[[60641, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60641', 200]]
[[114386, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114386', 500]]
[[56507, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56507', 200]]
[[63696, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63696', 200]]
[[57480, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57480', 200]]
[[63578, 'https://www.m

[[65610, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65610', 200]]
[[114346, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114346', 500]]
[[61375, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61375', 500]]
[[57126, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57126', 500]]
[[62352, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62352', 200]]
[[56412, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56412', 200]]
[[65857, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65857', 500]]
[[70711, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70711', 200]]
[[56598, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56598', 200]]
[[56322, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56322', 500]]
[[61762, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61762', 500]]
[[63646, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63646', 200]]
[[62341, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62341', 200]]
[[60868, 'https://www.m

[[63459, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63459', 200]]
[[69941, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69941', 200]]
[[62119, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62119', 200]]
[[60950, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60950', 500]]
[[70637, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70637', 200]]
[[114974, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114974', 500]]
[[63206, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63206', 200]]
[[57519, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57519', 500]]
[[56458, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56458', 200]]
[[61656, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61656', 500]]
[[63210, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63210', 200]]
[[63213, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63213', 200]]
[[61489, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61489', 500]]
[[58803, 'https://www.m

[[63525, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63525', 200]]
[[56371, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56371', 500]]
[[62594, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62594', 500]]
[[63661, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63661', 500]]
[[56663, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56663', 500]]
[[56072, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56072', 500]]
[[69991, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69991', 200]]
[[61199, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61199', 500]]
[[65894, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65894', 200]]
[[62980, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62980', 200]]
[[114347, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114347', 500]]
[[114177, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114177', 500]]
[[70590, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70590', 200]]
[[114920, 'https://ww

[[70705, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70705', 200]]
[[70723, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70723', 200]]
[[70143, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70143', 200]]
[[61206, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61206', 500]]
[[66067, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66067', 200]]
[[70495, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70495', 200]]
[[114626, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114626', 500]]
[[69976, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69976', 200]]
[[56362, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56362', 500]]
[[79361, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79361', 200]]
[[57515, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57515', 500]]
[[58550, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58550', 200]]
[[61724, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61724', 500]]
[[58443, 'https://www.m

[[62316, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62316', 200]]
[[66027, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66027', 200]]
[[62050, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62050', 500]]
[[60614, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60614', 200]]
[[114313, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114313', 500]]
[[60853, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60853', 200]]
[[65733, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65733', 200]]
[[63168, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63168', 200]]
[[60406, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60406', 500]]
[[56454, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56454', 200]]
[[69966, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69966', 200]]
[[58408, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58408', 500]]
[[62098, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62098', 200]]
[[70288, 'https://www.m

[[61951, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61951', 500]]
[[70188, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70188', 200]]
[[60409, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60409', 500]]
[[63259, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63259', 200]]
[[70757, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70757', 200]]
[[62906, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62906', 200]]
[[70090, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70090', 200]]
[[70318, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70318', 200]]
[[57031, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57031', 200]]
[[70489, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70489', 200]]
[[79534, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79534', 200]]
[[79372, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79372', 200]]
[[57415, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57415', 500]]
[[63309, 'https://www.med

[[56229, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56229', 200]]
[[79052, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79052', 500]]
[[62622, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62622', 500]]
[[69923, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69923', 200]]
[[62206, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62206', 500]]
[[61876, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61876', 200]]
[[69900, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69900', 200]]
[[56818, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56818', 200]]
[[79348, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79348', 200]]
[[114613, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114613', 500]]
[[63592, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63592', 200]]
[[63071, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63071', 200]]
[[60602, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60602', 200]]
[[57463, 'https://www.m

[[70010, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70010', 200]]
[[56780, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56780', 500]]
[[66045, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66045', 200]]
[[63162, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63162', 200]]
[[62220, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62220', 500]]
[[69986, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69986', 200]]
[[61600, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61600', 500]]
[[60474, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60474', 500]]
[[63606, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63606', 500]]
[[63037, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63037', 500]]
[[46506, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46506', 500]]
[[61349, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61349', 500]]
[[57308, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57308', 200]]
[[61457, 'https://www.med

[[57460, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57460', 500]]
[[57504, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57504', 200]]
[[57022, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57022', 500]]
[[114631, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114631', 500]]
[[56921, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56921', 200]]
[[79507, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79507', 200]]
[[63623, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63623', 200]]
[[58388, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58388', 500]]
[[70490, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70490', 200]]
[[79060, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79060', 500]]
[[70233, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70233', 200]]
[[46489, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46489', 500]]
[[61453, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61453', 500]]
[[61055, 'https://www.m

[[70379, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70379', 200]]
[[30425, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30425', 500]]
[[58394, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58394', 500]]
[[66104, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66104', 200]]
[[114510, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114510', 500]]
[[61799, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61799', 500]]
[[79219, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79219', 200]]
[[30423, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30423', 500]]
[[62914, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62914', 200]]
[[56585, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56585', 200]]
[[63121, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63121', 200]]
[[56603, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56603', 200]]
[[61617, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61617', 500]]
[[114078, 'https://www.

[[56220, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56220', 200]]
[[57386, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57386', 500]]
[[58596, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58596', 200]]
[[114194, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114194', 500]]
[[56845, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56845', 500]]
[[65834, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65834', 200]]
[[60959, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60959', 500]]
[[114989, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114989', 500]]
[[58563, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58563', 200]]
[[62121, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62121', 200]]
[[114619, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114619', 500]]
[[70572, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70572', 200]]
[[61143, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61143', 500]]
[[57024, 'https://w

[[60901, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60901', 500]]
[[63649, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63649', 200]]
[[114968, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114968', 500]]
[[61928, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61928', 500]]
[[61611, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61611', 500]]
[[63214, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63214', 200]]
[[56884, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56884', 500]]
[[62529, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62529', 200]]
[[65754, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65754', 500]]
[[79307, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79307', 200]]
[[61141, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61141', 500]]
[[114145, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114145', 500]]
[[61568, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61568', 500]]
[[63512, 'https://www

[[61435, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61435', 500]]
[[114679, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114679', 500]]
[[79248, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79248', 200]]
[[56720, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56720', 500]]
[[60767, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60767', 200]]
[[58751, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58751', 200]]
[[62559, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62559', 500]]
[[114942, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114942', 500]]
[[62152, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62152', 200]]
[[63610, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63610', 500]]
[[70336, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70336', 200]]
[[60733, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60733', 500]]
[[114445, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114445', 500]]
[[62517, 'https://w

[[63526, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63526', 200]]
[[114080, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114080', 500]]
[[56842, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56842', 500]]
[[70027, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70027', 200]]
[[63105, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63105', 200]]
[[58451, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58451', 500]]
[[56446, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56446', 200]]
[[61437, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61437', 500]]
[[70218, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70218', 200]]
[[61157, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61157', 500]]
[[56345, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56345', 500]]
[[63615, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63615', 500]]
[[114451, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114451', 500]]
[[61010, 'https://www

[[61119, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61119', 500]]
[[63135, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63135', 200]]
[[60568, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60568', 500]]
[[70080, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70080', 200]]
[[66209, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66209', 200]]
[[79375, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79375', 200]]
[[70173, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70173', 200]]
[[56701, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56701', 500]]
[[66228, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66228', 200]]
[[62929, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62929', 200]]
[[114976, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114976', 500]]
[[79232, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79232', 200]]
[[60429, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60429', 200]]
[[56736, 'https://www.m

[[60555, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60555', 500]]
[[79197, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79197', 200]]
[[57347, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57347', 200]]
[[70718, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70718', 200]]
[[57210, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57210', 200]]
[[61476, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61476', 500]]
[[114331, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114331', 500]]
[[65972, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65972', 500]]
[[62563, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62563', 500]]
[[114739, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114739', 500]]
[[70329, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70329', 200]]
[[60802, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60802', 200]]
[[61677, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61677', 500]]
[[114799, 'https://ww

[[70412, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70412', 200]]
[[65542, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65542', 500]]
[[79319, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79319', 200]]
[[63488, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63488', 500]]
[[114635, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114635', 500]]
[[70513, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70513', 200]]
[[60427, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60427', 500]]
[[114329, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114329', 500]]
[[70656, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70656', 200]]
[[114858, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114858', 500]]
[[56995, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56995', 200]]
[[58495, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58495', 200]]
[[61441, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61441', 500]]
[[66253, 'https://w

[[56396, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56396', 200]]
[[61559, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61559', 500]]
[[46514, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46514', 500]]
[[62669, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62669', 500]]
[[63362, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63362', 500]]
[[63294, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63294', 500]]
[[58539, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58539', 200]]
[[60651, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60651', 500]]
[[60854, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60854', 200]]
[[70268, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70268', 200]]
[[61017, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61017', 500]]
[[63239, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63239', 200]]
[[65593, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65593', 500]]
[[57499, 'https://www.med

[[62409, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62409', 200]]
[[63527, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63527', 200]]
[[56461, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56461', 200]]
[[58772, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58772', 200]]
[[61663, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61663', 500]]
[[57389, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57389', 500]]
[[56416, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56416', 200]]
[[63690, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63690', 200]]
[[57117, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57117', 200]]
[[55996, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=55996', 500]]
[[62657, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62657', 500]]
[[61273, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61273', 500]]
[[70414, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70414', 200]]
[[62739, 'https://www.med

[[70508, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70508', 200]]
[[56027, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56027', 500]]
[[60852, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60852', 200]]
[[62216, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62216', 500]]
[[65753, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65753', 500]]
[[114720, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114720', 500]]
[[114927, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114927', 500]]
[[60632, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60632', 500]]
[[57188, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57188', 200]]
[[56152, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56152', 200]]
[[56198, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56198', 500]]
[[62308, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62308', 200]]
[[66176, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66176', 200]]
[[65649, 'https://www

[[65952, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65952', 200]]
[[45897, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=45897', 200]]
[[65986, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65986', 200]]
[[57500, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57500', 200]]
[[70052, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70052', 200]]
[[60966, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60966', 500]]
[[56709, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56709', 500]]
[[56700, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56700', 500]]
[[62776, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62776', 500]]
https://www.medregom.admin.ch/DE/Detail/Detail?pid=61012 Scraping paused
[[62194, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62194', 500]]
[[58626, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58626', 200]]
[[65974, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65974', 200]]
[[57248, 'https://www.medre

[[56710, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56710', 500]]
[[57106, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57106', 200]]
[[58583, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58583', 200]]
[[60779, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60779', 500]]
[[62114, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62114', 200]]
[[61047, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61047', 200]]
[[62998, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62998', 200]]
[[46451, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46451', 500]]
[[56361, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56361', 200]]
[[61092, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61092', 500]]
[[58695, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58695', 200]]
[[114688, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114688', 500]]
[[62487, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62487', 200]]
[[63517, 'https://www.m

[[62786, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62786', 500]]
[[63005, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63005', 500]]
[[63208, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63208', 200]]
[[114926, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114926', 500]]
[[62680, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62680', 500]]
[[62761, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62761', 200]]
[[63454, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63454', 200]]
[[56866, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56866', 500]]
[[58663, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58663', 200]]
[[56357, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56357', 500]]
[[57255, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57255', 200]]
[[56089, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56089', 200]]
[[62693, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62693', 200]]
[[70680, 'https://www.m

[[62327, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62327', 200]]
[[114124, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114124', 500]]
[[60882, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60882', 500]]
[[70134, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70134', 200]]
[[63019, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63019', 200]]
[[63471, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63471', 200]]
[[114779, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114779', 500]]
[[61844, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61844', 500]]
[[62553, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62553', 500]]
[[65943, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65943', 500]]
[[69913, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69913', 200]]
[[63176, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63176', 200]]
[[57383, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57383', 200]]
[[61659, 'https://www

[[79569, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79569', 200]]
[[79388, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79388', 200]]
[[63539, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63539', 200]]
[[65832, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65832', 200]]
[[69973, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69973', 200]]
[[58397, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58397', 500]]
[[62304, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62304', 200]]
[[114774, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114774', 500]]
[[69974, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69974', 200]]
[[70044, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70044', 200]]
[[56400, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56400', 200]]
[[70717, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70717', 200]]
[[70162, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70162', 500]]
[[57100, 'https://www.m

[[60612, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60612', 500]]
[[58505, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58505', 200]]
[[63625, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63625', 200]]
[[79136, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79136', 200]]
[[56404, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56404', 200]]
[[70049, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70049', 200]]
[[63229, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63229', 200]]
[[114548, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114548', 500]]
[[56902, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56902', 500]]
[[114684, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114684', 500]]
[[63679, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63679', 200]]
[[114054, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114054', 500]]
[[62697, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62697', 200]]
[[56321, 'https://w

[[62601, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62601', 500]]
[[56233, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56233', 200]]
[[30390, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30390', 500]]
[[61418, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61418', 200]]
[[56301, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56301', 500]]
[[114264, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114264', 500]]
[[56762, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56762', 500]]
[[65613, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65613', 200]]
[[62465, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62465', 200]]
[[65720, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65720', 200]]
[[62444, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62444', 200]]
[[63289, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63289', 500]]
[[57487, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57487', 500]]
[[62833, 'https://www.m

[[62924, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62924', 200]]
[[114831, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114831', 500]]
[[57508, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57508', 200]]
[[62251, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62251', 200]]
[[63637, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63637', 200]]
[[70621, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70621', 200]]
[[61405, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61405', 500]]
[[69896, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69896', 200]]
[[114501, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114501', 500]]
[[114953, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114953', 500]]
[[62933, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62933', 200]]
[[79110, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79110', 200]]
[[66017, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66017', 200]]
[[66227, 'https://w

[[62167, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62167', 500]]
[[66141, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66141', 200]]
[[65997, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65997', 200]]
[[70429, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70429', 200]]
[[114909, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114909', 500]]
[[56242, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56242', 200]]
[[63249, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63249', 500]]
[[62060, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62060', 200]]
[[62237, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62237', 200]]
[[57457, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57457', 500]]
[[56325, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56325', 500]]
[[62318, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62318', 200]]
[[62136, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62136', 200]]
[[46512, 'https://www.m

[[63349, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63349', 500]]
[[66240, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66240', 200]]
[[61974, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61974', 500]]
[[56959, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56959', 200]]
[[61305, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61305', 500]]
[[61102, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61102', 500]]
[[70133, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70133', 200]]
[[58786, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58786', 200]]
[[60534, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60534', 200]]
[[56128, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56128', 500]]
[[57144, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57144', 200]]
[[114590, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114590', 500]]
[[79351, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79351', 200]]
[[56888, 'https://www.m

[[65788, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65788', 500]]
[[60843, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60843', 200]]
[[79295, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79295', 200]]
[[63334, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63334', 500]]
[[63536, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63536', 200]]
[[60516, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60516', 500]]
[[62424, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62424', 200]]
[[63291, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63291', 500]]
[[63469, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63469', 200]]
[[57181, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57181', 200]]
[[61992, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61992', 500]]
[[65748, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65748', 200]]
[[62579, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62579', 500]]
[[114257, 'https://www.me

[[114896, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114896', 500]]
[[56037, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56037', 500]]
[[60819, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60819', 200]]
[[56906, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56906', 200]]
[[58737, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58737', 200]]
[[56080, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56080', 200]]
[[61675, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61675', 500]]
[[79353, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79353', 200]]
[[58578, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58578', 200]]
[[56830, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56830', 500]]
[[62051, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62051', 200]]
[[114963, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114963', 500]]
[[57399, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57399', 500]]
[[66153, 'https://www

[[70758, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70758', 200]]
[[56987, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56987', 200]]
[[56993, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56993', 200]]
[[62818, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62818', 200]]
[[58454, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58454', 200]]
[[57311, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57311', 500]]
[[114948, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114948', 500]]
[[63513, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63513', 200]]
[[60503, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60503', 500]]
[[62570, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62570', 500]]
[[61962, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61962', 500]]
[[62084, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62084', 200]]
[[56595, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56595', 200]]
[[58817, 'https://www.m

[[61353, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61353', 500]]
[[114674, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114674', 500]]
[[60954, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60954', 500]]
[[61169, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61169', 200]]
[[65885, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65885', 200]]
[[63310, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63310', 200]]
[[60834, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60834', 200]]
[[79498, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79498', 200]]
[[79461, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79461', 200]]
[[79115, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79115', 200]]
[[79456, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79456', 200]]
[[65586, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65586', 200]]
[[56593, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56593', 200]]
[[56855, 'https://www.m

[[62634, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62634', 500]]
[[56289, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56289', 200]]
[[58510, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58510', 200]]
[[69936, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69936', 200]]
[[57340, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57340', 500]]
[[114472, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114472', 500]]
[[60841, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60841', 200]]
[[62985, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62985', 200]]
[[57514, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57514', 200]]
[[70234, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70234', 200]]
[[70411, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70411', 200]]
[[56815, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56815', 500]]
[[60484, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60484', 500]]
[[56165, 'https://www.m

[[57331, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57331', 500]]
[[79414, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79414', 200]]
[[62022, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62022', 500]]
[[61380, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61380', 500]]
[[62065, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62065', 200]]
[[70245, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70245', 200]]
[[56302, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56302', 500]]
[[65789, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65789', 500]]
[[60505, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60505', 500]]
[[65591, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65591', 200]]
[[62645, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62645', 500]]
[[56159, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56159', 500]]
[[66048, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66048', 200]]
[[58477, 'https://www.med

[[69920, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69920', 200]]
[[56991, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56991', 200]]
[[70018, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70018', 200]]
[[63483, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63483', 500]]
[[62526, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62526', 200]]
[[62950, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62950', 500]]
[[61065, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61065', 500]]
[[66188, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66188', 200]]
[[66216, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66216', 200]]
[[61828, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61828', 500]]
[[114266, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114266', 500]]
[[30399, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30399', 500]]
[[60709, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60709', 500]]
[[65858, 'https://www.m

[[61301, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61301', 500]]
[[62687, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62687', 200]]
[[60571, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60571', 500]]
[[70113, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70113', 200]]
[[58456, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58456', 200]]
[[58795, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58795', 200]]
[[58849, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58849', 200]]
[[61389, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61389', 500]]
[[62005, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62005', 500]]
[[61275, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61275', 500]]
[[61414, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61414', 500]]
[[114367, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114367', 500]]
[[66208, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66208', 200]]
[[46529, 'https://www.m

[[62377, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62377', 200]]
[[63065, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63065', 500]]
[[61908, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61908', 500]]
[[56833, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56833', 200]]
[[57441, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57441', 500]]
[[56304, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56304', 500]]
[[70157, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70157', 200]]
[[70248, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70248', 200]]
[[56653, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56653', 500]]
[[56009, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56009', 200]]
[[60831, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60831', 200]]
[[70556, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70556', 200]]
[[63657, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63657', 200]]
[[61513, 'https://www.med

[[57224, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57224', 200]]
[[56537, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56537', 200]]
[[57337, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57337', 200]]
[[61834, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61834', 500]]
[[61008, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61008', 500]]
[[63144, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63144', 200]]
[[56264, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56264', 200]]
[[62193, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62193', 500]]
[[61667, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61667', 500]]
[[114718, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114718', 500]]
[[114849, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114849', 500]]
[[56712, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56712', 500]]
[[62491, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62491', 200]]
[[56443, 'https://www

[[56556, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56556', 200]]
[[70319, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70319', 200]]
[[65891, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65891', 200]]
[[65685, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65685', 200]]
[[66236, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66236', 200]]
[[69999, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69999', 200]]
[[56664, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56664', 500]]
[[65757, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65757', 500]]
[[79477, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79477', 200]]
[[114908, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114908', 500]]
[[65811, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65811', 200]]
[[79483, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79483', 200]]
[[62228, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62228', 200]]
[[60990, 'https://www.m

[[114129, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114129', 500]]
[[65721, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65721', 500]]
[[65768, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65768', 200]]
[[65706, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65706', 500]]
[[58683, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58683', 200]]
[[62319, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62319', 200]]
[[65568, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65568', 200]]
[[60724, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60724', 500]]
[[79341, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79341', 200]]
[[69905, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69905', 200]]
[[62037, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62037', 500]]
[[79087, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79087', 200]]
[[65739, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65739', 500]]
[[57110, 'https://www.m

[[61574, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61574', 500]]
[[60879, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60879', 500]]
[[61271, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61271', 500]]
[[62569, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62569', 500]]
[[79502, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79502', 200]]
[[56761, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56761', 500]]
[[62917, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62917', 200]]
[[61964, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61964', 500]]
[[69977, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69977', 200]]
[[61654, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61654', 500]]
[[61274, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61274', 500]]
[[79496, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79496', 200]]
[[60856, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60856', 200]]
[[70413, 'https://www.med

[[60414, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60414', 500]]
[[56074, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56074', 200]]
[[79450, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79450', 200]]
[[65544, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65544', 200]]
[[70448, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70448', 200]]
[[30422, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30422', 500]]
[[69997, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69997', 200]]
[[62190, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62190', 500]]
[[56147, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56147', 200]]
[[70450, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70450', 200]]
[[57139, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57139', 500]]
[[114263, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114263', 500]]
[[58701, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58701', 200]]
[[79201, 'https://www.m

[[65800, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65800', 500]]
[[70608, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70608', 200]]
[[63506, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63506', 200]]
[[62168, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62168', 500]]
[[61446, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61446', 500]]
[[56619, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56619', 200]]
[[79357, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79357', 200]]
[[70341, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70341', 200]]
[[62773, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62773', 500]]
[[61411, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61411', 500]]
[[57029, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57029', 200]]
[[79459, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79459', 200]]
[[60961, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60961', 500]]
[[56567, 'https://www.med

[[114918, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114918', 500]]
[[114645, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114645', 500]]
[[66165, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66165', 200]]
[[56606, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56606', 200]]
[[62110, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62110', 200]]
[[61637, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61637', 500]]
[[62796, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62796', 200]]
[[79548, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79548', 200]]
[[57438, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57438', 500]]
[[60703, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60703', 500]]
[[70376, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70376', 200]]
[[60884, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60884', 500]]
[[114515, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114515', 500]]
[[62373, 'https://w

[[61978, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61978', 500]]
[[56189, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56189', 500]]
[[66191, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66191', 200]]
[[57154, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57154', 200]]
[[61563, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61563', 500]]
[[65710, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65710', 200]]
[[79536, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79536', 200]]
[[63647, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63647', 200]]
[[57436, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57436', 200]]
[[61262, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61262', 500]]
[[66193, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66193', 200]]
[[63475, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63475', 200]]
[[65549, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65549', 200]]
[[70140, 'https://www.med

[[56149, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56149', 200]]
[[63566, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63566', 200]]
[[79095, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79095', 200]]
[[63236, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63236', 200]]
[[62664, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62664', 500]]
[[63365, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63365', 500]]
[[62715, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62715', 200]]
[[70682, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70682', 200]]
[[60494, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60494', 500]]
[[56055, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56055', 500]]
[[56872, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56872', 200]]
[[63389, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63389', 200]]
[[56725, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56725', 500]]
[[66034, 'https://www.med

[[114034, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114034', 500]]
[[56850, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56850', 200]]
[[66161, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66161', 200]]
[[114383, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114383', 500]]
[[70570, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70570', 200]]
[[57358, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57358', 200]]
[[63537, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63537', 200]]
[[61650, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61650', 500]]
[[56423, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56423', 200]]
[[114657, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114657', 500]]
[[62376, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62376', 200]]
[[57244, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57244', 500]]
[[58561, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58561', 200]]
[[58493, 'https://w

[[56088, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56088', 500]]
[[62734, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62734', 200]]
[[60530, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60530', 500]]
[[70410, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70410', 200]]
[[63587, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63587', 200]]
[[46523, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46523', 500]]
[[114689, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114689', 500]]
[[70369, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70369', 200]]
[[79419, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79419', 200]]
[[56586, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56586', 200]]
[[56299, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56299', 200]]
[[114128, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114128', 500]]
[[63456, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63456', 200]]
[[62483, 'https://www

[[79158, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79158', 200]]
[[61702, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61702', 500]]
[[70033, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70033', 200]]
[[56893, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56893', 200]]
[[62763, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62763', 500]]
[[66030, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66030', 200]]
[[62835, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62835', 200]]
[[56838, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56838', 500]]
[[63273, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63273', 500]]
[[65550, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65550', 200]]
[[56150, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56150', 200]]
[[79255, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79255', 200]]
[[61797, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61797', 500]]
[[114378, 'https://www.me

[[114162, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114162', 500]]
[[61555, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61555', 500]]
[[60758, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60758', 500]]
[[60631, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60631', 500]]
[[46531, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46531', 500]]
[[69963, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69963', 200]]
[[46484, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46484', 500]]
[[69952, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69952', 200]]
[[63403, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63403', 200]]
[[114964, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114964', 500]]
[[56641, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56641', 500]]
[[56332, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56332', 500]]
[[57430, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57430', 500]]
[[63258, 'https://www

[[63555, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63555', 200]]
[[70418, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70418', 200]]
[[62362, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62362', 200]]
[[56235, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56235', 200]]
[[55986, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=55986', 200]]
[[62587, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62587', 500]]
[[70740, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70740', 200]]
[[58522, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58522', 200]]
[[57249, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57249', 500]]
[[63266, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63266', 500]]
[[70456, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70456', 200]]
[[57088, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57088', 200]]
[[79053, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79053', 500]]
[[63560, 'https://www.med

[[79196, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79196', 200]]
[[61302, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61302', 500]]
[[65731, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65731', 200]]
[[63185, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63185', 200]]
[[70214, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70214', 200]]
[[62522, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62522', 200]]
[[114090, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114090', 500]]
[[79325, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79325', 200]]
[[63515, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63515', 200]]
[[57202, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57202', 200]]
[[57421, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57421', 200]]
[[114185, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114185', 500]]
[[79492, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79492', 200]]
[[62764, 'https://www

[[70653, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70653', 200]]
[[58747, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58747', 200]]
[[70361, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70361', 200]]
[[69942, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69942', 200]]
[[57429, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57429', 500]]
[[61160, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61160', 500]]
[[63335, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63335', 500]]
[[56616, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56616', 200]]
[[70364, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70364', 200]]
[[79350, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79350', 200]]
[[114273, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114273', 500]]
[[58722, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58722', 200]]
[[65896, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65896', 200]]
[[61674, 'https://www.m

[[65839, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65839', 200]]
[[60483, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60483', 500]]
[[79105, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79105', 200]]
[[66220, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66220', 200]]
[[62082, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62082', 200]]
[[62464, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62464', 200]]
[[61768, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61768', 500]]
[[63381, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63381', 500]]
[[63711, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63711', 200]]
[[114747, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114747', 500]]
[[79154, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79154', 500]]
[[63345, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63345', 500]]
[[79544, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79544', 200]]
[[63405, 'https://www.m

[[56356, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56356', 200]]
[[79164, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79164', 200]]
[[61815, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61815', 500]]
[[62504, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62504', 200]]
[[62185, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62185', 500]]
[[114552, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114552', 500]]
[[60929, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60929', 500]]
[[66221, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66221', 200]]
[[66012, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66012', 200]]
[[63218, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63218', 200]]
[[56143, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56143', 200]]
[[57404, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57404', 500]]
[[61331, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61331', 500]]
[[70160, 'https://www.m

[[58657, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58657', 200]]
[[61937, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61937', 500]]
[[79242, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79242', 200]]
[[70663, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70663', 200]]
[[60895, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60895', 500]]
[[114996, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114996', 500]]
[[58569, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58569', 200]]
[[56579, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56579', 200]]
[[56957, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56957', 200]]
[[79313, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79313', 200]]
[[79403, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79403', 200]]
[[57257, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57257', 200]]
[[57116, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57116', 200]]
[[63518, 'https://www.m

[[62105, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62105', 200]]
[[63230, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63230', 200]]
[[61696, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61696', 500]]
[[63202, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63202', 200]]
[[62207, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62207', 500]]
[[61712, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61712', 200]]
[[66179, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66179', 200]]
[[79368, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79368', 200]]
[[63481, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63481', 500]]
[[66256, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66256', 200]]
[[66102, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66102', 200]]
[[56976, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56976', 500]]
[[57108, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57108', 200]]
[[61603, 'https://www.med

[[63166, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63166', 200]]
[[60936, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60936', 500]]
[[55991, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=55991', 200]]
[[30391, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30391', 500]]
[[62199, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62199', 500]]
[[79282, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79282', 200]]
[[114151, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114151', 500]]
[[58428, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58428', 200]]
[[114691, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114691', 500]]
[[56492, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56492', 200]]
[[56979, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56979', 200]]
[[79474, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79474', 200]]
[[57245, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57245', 500]]
[[60983, 'https://www

[[62087, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62087', 200]]
[[65855, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65855', 500]]
[[66171, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66171', 200]]
[[65765, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65765', 200]]
[[56470, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56470', 200]]
[[114213, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114213', 500]]
[[61579, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61579', 500]]
[[56193, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56193', 200]]
[[61691, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61691', 500]]
[[79385, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79385', 200]]
[[62456, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62456', 200]]
[[58757, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58757', 200]]
[[58748, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58748', 200]]
[[56137, 'https://www.m

[[62623, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62623', 500]]
[[57035, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57035', 200]]
[[79278, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79278', 200]]
[[57462, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57462', 500]]
https://www.medregom.admin.ch/DE/Detail/Detail?pid=65585 Scraping paused
[[61836, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61836', 500]]
[[70061, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70061', 200]]
[[79558, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79558', 200]]
[[114732, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114732', 500]]
[[70332, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70332', 200]]
[[61410, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61410', 500]]
[[60575, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60575', 500]]
[[69886, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69886', 200]]
[[60894, 'https://www.med

[[63293, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63293', 500]]
[[114910, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114910', 500]]
[[61050, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61050', 500]]
[[114031, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114031', 500]]
[[56094, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56094', 200]]
[[61753, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61753', 500]]
[[61229, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61229', 500]]
[[61303, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61303', 500]]
[[61932, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61932', 500]]
[[61116, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61116', 500]]
[[63413, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63413', 200]]
[[66011, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66011', 200]]
[[60789, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60789', 200]]
[[56280, 'https://www

[[70148, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70148', 200]]
[[70574, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70574', 200]]
[[62828, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62828', 200]]
[[61642, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61642', 500]]
[[60887, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60887', 500]]
[[70067, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70067', 200]]
[[79089, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79089', 200]]
[[62957, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62957', 500]]
[[62177, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62177', 500]]
[[65772, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65772', 200]]
[[61324, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61324', 500]]
[[56988, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56988', 500]]
[[69927, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69927', 200]]
[[61481, 'https://www.med

[[79565, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79565', 200]]
[[56197, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56197', 500]]
[[63268, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63268', 500]]
[[70525, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70525', 200]]
[[58718, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58718', 200]]
[[56487, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56487', 200]]
[[66037, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66037', 200]]
[[60586, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60586', 200]]
[[114400, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114400', 500]]
[[61906, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61906', 500]]
[[57412, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57412', 500]]
[[62461, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62461', 200]]
[[62619, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62619', 500]]
[[62347, 'https://www.m

[[56261, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56261', 200]]
[[58832, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58832', 200]]
[[79451, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79451', 200]]
[[114707, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114707', 500]]
[[114821, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114821', 500]]
[[62900, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62900', 500]]
[[57032, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57032', 500]]
[[65634, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65634', 500]]
[[114607, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114607', 500]]
[[66078, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66078', 200]]
[[63338, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63338', 200]]
[[60898, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60898', 500]]
[[57270, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57270', 500]]
[[60610, 'https://w

[[70092, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70092', 200]]
[[114745, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114745', 500]]
[[61979, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61979', 500]]
[[61772, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61772', 500]]
[[114856, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114856', 500]]
[[60694, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60694', 500]]
[[70274, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70274', 200]]
[[61761, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61761', 500]]
[[46500, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46500', 200]]
[[70029, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70029', 200]]
[[70460, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70460', 200]]
[[61138, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61138', 500]]
[[56940, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56940', 200]]
[[70304, 'https://www

[[61052, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61052', 500]]
[[114727, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114727', 500]]
[[56811, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56811', 500]]
[[79249, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=79249', 200]]
[[63705, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63705', 200]]
[[65968, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65968', 200]]
[[57231, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57231', 500]]
[[70708, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70708', 200]]
[[63499, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63499', 200]]
[[63618, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63618', 500]]
[[56499, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56499', 200]]
[[56090, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56090', 200]]
[[56285, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56285', 200]]
[[57478, 'https://www.m

[[58435, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58435', 200]]
[[61692, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61692', 500]]
[[114146, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114146', 500]]
[[61960, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61960', 500]]
[[61713, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61713', 500]]
[[61934, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61934', 500]]
[[62576, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62576', 500]]
[[70118, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70118', 200]]
[[70262, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70262', 200]]
[[57253, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57253', 200]]
[[65697, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65697', 200]]
[[56225, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56225', 200]]
[[63079, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63079', 200]]
[[57030, 'https://www.m

[[65927, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65927', 200]]
[[58698, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58698', 200]]
[[56082, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56082', 200]]
[[69959, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69959', 200]]
[[62241, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62241', 200]]
[[114252, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114252', 500]]
[[60428, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60428', 500]]
[[63313, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63313', 200]]
[[58637, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=58637', 200]]
[[56753, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56753', 500]]
[[60814, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60814', 200]]
[[62780, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62780', 500]]
[[65980, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65980', 200]]
[[56643, 'https://www.m

[[60440, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60440', 500]]
[[63434, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63434', 200]]
[[62942, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62942', 200]]
[[56763, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56763', 500]]
[[70498, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70498', 200]]
[[66185, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66185', 200]]
[[114608, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114608', 500]]
[[114172, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114172', 500]]
[[114914, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114914', 500]]
[[61304, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61304', 500]]
[[61287, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61287', 500]]
[[114260, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114260', 500]]
[[61976, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61976', 500]]
[[61863, 'https:/

[[62445, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62445', 200]]
[[61320, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61320', 500]]
[[61841, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61841', 500]]
[[46504, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46504', 500]]
[[57367, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57367', 500]]
[[69979, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=69979', 200]]
[[60481, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60481', 500]]
[[70434, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70434', 200]]
[[56296, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56296', 500]]
[[70008, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70008', 200]]
[[62059, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62059', 200]]
[[46485, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46485', 500]]
[[56582, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56582', 200]]
[[61631, 'https://www.med

[[30398, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=30398', 500]]
[[114705, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114705', 500]]
[[70177, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70177', 200]]
[[56281, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56281', 500]]
[[62034, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62034', 500]]
[[57280, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=57280', 500]]
[[46525, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=46525', 200]]
[[61245, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61245', 500]]
[[114843, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=114843', 500]]
[[63448, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=63448', 200]]
[[56835, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56835', 200]]
[[70108, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=70108', 200]]
[[66036, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=66036', 200]]
[[114291, 'https://ww

Wir haben 8339 Files runtergeladen, es fehlen also noch 6 Files, vermutlich aus den Unterbrüchen. Mit einer manuellen Suche um Printout suchen wir die entsprechenden Nummern. 
Es sind: 62899, 60446, 62596, 61012, 56565, 65585


In [40]:
url_paused = [62899, 60446, 62596, 61012, 56565, 65585]



[62899, 60446, 62596, 61012, 56565, 65585]

In [43]:
url_stamm = "https://www.medregom.admin.ch/DE/Detail/Detail?pid="
output_path = "medreg_rescrape2/"
headers = {'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36'}

for url_id in url_paused: #die Liste der Files die beim Pausieren ausgelassen wurden
    url_req = url_stamm + str(url_id) 
    r = requests.get(url_req, headers = headers, timeout=20)

    with open(output_path + "medreg_" + str(url_id) + ".html", 'w', encoding = 'utf-8') as f:
        f.write(r.text)
        f.close()
        protokoll = [[url_id, url_req, r.status_code]]
        print(protokoll)
        

[[62899, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62899', 500]]
[[60446, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=60446', 500]]
[[62596, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=62596', 500]]
[[61012, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=61012', 500]]
[[56565, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=56565', 200]]
[[65585, 'https://www.medregom.admin.ch/DE/Detail/Detail?pid=65585', 200]]


Ok, jetzt haben wir aus dem Re-Scraping 2 8345 File nochmals runtergeladen. 

Check: sind noch Javascript-Meldungen darunter? Nein! 

### Es scheint endlich soweit: wir sollten eigentlich jetzt alle Daten haben.

Wir führen jetzt alle Files aus dem Ordner medreg_rescrape2 im Ordner medreg_2020_scraped_data zusammen. Resultat: Wir haben jetzt 114799 Files im Ordner. 

Wir checken nochmals die Files durch. Die Content-Suche nach *Ärztin/Arzt* ergibt jetzt 76818 Treffer. Dieser Wert ist schlüssig. Durch das Re-Scraping am 19.1.2020 wurden wohl auch neue Einträge erfasst, die beim ersten Scraping noch nicht vorhanden waren. 

Zum Vergleich: am 14.1. umfasst die Datenbank für Ärztin/Arzt 76765, Chriropraktor 386, Zahnarzt 11997, Apotheker 10999, Tierarzt 5315. 
Am 19.1.2020 zeigt die Hauptseite 76820 Einträge für Ärztin/Arzt an. 

Wir wollen diese Datensätze jetzt in einen neuen Ordner *medreg_2020_doctors* transferieren. Dabei zeigt sich erneut, dass in MacOS mit dem Handling von sovielen Daten an die Grenze kommt. Das Kopieren ist per Drag und Drop nicht auf einmal möglich. Das Macbook arbeitet ständig mit Lüfter. Der Finder reagiert während Minuten nicht mehr. 
Es geht leider dabei viel Zeit verloren. Versuche mit alternativen Filemanagern (Nimble Commander, Commander One, PathFinder) scheitern. Nach zum Teil minutemlangen Warten mit gedrückter Maustaste breche ich die Kopierversuche ab. 

Wir konzentrieren uns also zunächst auf die Suche nach Psychiatern und legen diese im Ordner *megreg_2020_psych* ab. Die Suche mit EasyFind ergibt 5440 Treffer (11 mehr als Stand MedReg 19.1.2020). Auch das ist schon mühsam. MacOS scheint nicht für die Arbeit mit vielen Files gemacht zu sein. 

Zudem: die Spotlight-Inhaltsuche liefert völlig falsche Daten. Die Suche nach Ärztin/Arzt ergibt zum Beispiel nur 218 Treffen, statt 76820 wie bei EasyFind. 


### Weitere Analysen der abgelegten Files erfolgen nun im Notebook MedReg_Parsing 04